src/app/map/map.component.ts

```diff
@@ -202,13 +202,13 @@ export class MapComponent implements OnInit {
                 if (o.text === "Query by Basin" && o.selected === true) {
                   this.addLayers('NHD Flowlines');
                   this.addLayers('Archived WildFire Perimeters');
```

Comment:
Can you please rename to "Archived Wildfire Perimeters" in the config.json (and elsewhere in the code)?